mqtt/mqtt-broker/src/auth/authorization.rs

```diff
@@ -66,13 +66,13 @@ pub struct AllowAll;
 impl Authorizer for AllowAll {
     type Error = Infallible;
 
-    fn authorize(&self, _: Activity) -> Result<Authorization, Self::Error> {
+    fn authorize(&self, _: &Activity) -> Result<Authorization, Self::Error> {
         Ok(Authorization::Allowed)
     }
 }
 
 /// Describes a client activity to authorized.
-#[derive(Debug)]
+#[derive(Clone, Debug)]
```

Comment:
I use it in one single place `PolicyAuthorizer.authorize`, b/c I need to pass a clone of the `Activity` to a policy `Request`. It can probably be improved, but I'll try to do that in the next PR.